src/components/tooltip/Tooltip.tsx

```diff
@@ -142,68 +134,79 @@ const Tooltip: React.FC<TooltipProps> = ({
     ],
   });
 
-  const children = <span>{otherProps.children}</span>;
-
   const isVisible =
     typeof visible !== 'undefined'
       ? visible
       : displayTrigger === 'hover'
         ? showHover
         : showClick;
 
+  const children = <div
+    className={ classnames('tk-tooltip__wrapper', wrapperClassName) }
```

Comment:
The `tk-tooltip_wrapper` does not exist in uiToolkit. I previously tried to expose something like `wrapperClassName` but @symphony-adnane mentioned that it's better not to expose it. So we went for a trick like `className={className ? className + '_wrapper' : null}`, this way you won't need to expose a wrapper classname. You can just add your wrapper classname following the format `classname + '_wrapper'` in your `style.ts`.

Reply:
I don't think I'm a big fan of the trick. If you need to style the wrapper, you need to know how to do it. It shouldn't be hidden, it's a part of this components interface.

Now it's not a very elegant solution but I think until something better can be thought of we should expose this className wrapper.

Reply:
wrapper classname to me seems quite an overkill of a prop...
Those are internals of the components IMO we'd be adding a prop for each subcomponent? (which are implementation details that may change in the future...) 
IMHO I'd suggest to use a custom class and append to it __wrapper, and document it 
I let you decide on that

Reply:
After discussion we said it's more clear the implementation with two different classNames. It's not ideal but works for now.